web/src/components/ProcessView.tsx

```diff
@@ -1,13 +1,17 @@
-import { AxiosResponse, AxiosError } from "axios";
+import Axios, { AxiosResponse, AxiosError } from "axios";
 import React, { useContext, useState, useEffect, FC } from "react";
 import "react-table/react-table.css";
-import { Graphviz } from "graphviz-react";
 
 import { AppContext, IAppContext } from "app/AppContext";
 import { api, ProcessData, StepData, RelationData } from "services/HaskellApiService";
 
 import "components/Graphviz.scss";
 
+import axiosErrorExceptionHandler from "./utils/axios_errors_handlers/AxiosErrorHander";
```

Comment:
The snake case here looks too artificial. Please use camelCase.

Or, if you put both handlers in one file, you can directly put this file in utils.